README.md

```diff
@@ -1,107 +1,105 @@
-# Application Template
-This is a template for you to start working within a Java environment. _Substitute this description for a description of your application_.
-**_Please feel free to add new sections or delete not-used sections_**.
+# Angular Component Update Playground
```

Comment:
Remember, Component update is just the first issue we are addressing, the Repository objective is much bigger than this.